src/iiwa_plan_manager.cc

```diff
@@ -154,7 +159,8 @@ void IiwaPlanManager::HandleIiwaStatus(
   bool command_has_error;
   {
     std::lock_guard<std::mutex> lock(mutex_state_machine_);
-    command_has_error = state_machine_->CommandHasError(s, c);
+    command_has_error = state_machine_->CommandHasError(s, c,
+      config_["q_threshold"].as<double>());
```

Comment:
I didn't want to route the config files through all the state machine stuff, so I ended up just having a threshold as an argument to the error function.